devel/python/python/ert/cwrap/basecclass.py

```diff
@@ -109,3 +109,36 @@ def __del__(self):
                 # we can have a Python object with c_pointer == None.
                 if self.__c_pointer > 0:
                     self.free()
+
+                    
+        
+    def dropCPointer(self):
```

Comment:
I suggest prefixing with underscore. Could this be called `_invalidate` instead? Then the long comment is rather unnecessary...